check-and-talk/src/components/SignIn.css

```
@@ -0,0 +1,64 @@
+*{font-family:Pretendard;}
+
+body{
+    margin: 0;
+}
+
+.leftScreen{
+    width: 50%;
+    height: 100vh;
+    float: left;
+    background-color: #464646;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    overflow: hidden;
+}
+
+.rightScreen{
+    width: 50%;
+    height: 100vh;
+    float: right;
+    background-color: #333333;
+    position: relative;
+    justify-content: center;
+}
+
+.Title{       
+    color:white ;
+    text-align: center;
+    font-size: 20pt;
+    margin-top: 15%;
+    margin-bottom: 8%;
+
+}
+
+.Google img, .Facebook img, .Github img {
+    width: 8%;
+    height: 8%;
+    padding-right: 40px;
+    padding-left: 40px;
+    text-align: center;
+}
+
+.Google span, .Github span, .Facebook span {
+    color: white;
```

```suggestion
    color: rgba(255, 255, 255, 0.5);
```

색을 좀 어둡게 하는게, 로고의 흰색과 구분되어 좋을 것 같아요

---

저도 동의합니다! 색을 어둡게 하거나 투명도를 주는 게 더 잘 구분될 것 같습니다.